### PR TITLE
Fix introspection for Servers expecting Post

### PR DESCRIPTION
**Pull request by Birnsen**
It is possible that a Get Request is successful
but does not contain a proper schema.

With this commit if a Get Requests succeds it
will be checked if a schema can be deserialized.
If not a Post request is taking place and also
checked for a proper schema.

### CODE REVIEW

File: src/FSharp.Data.GraphQL.Client/GraphQLClient.fs [hunk 1]
```diff
@@ -88,7 +88,7 @@ module GraphQLClient =
         |> Async.RunSynchronously
 
     /// Executes an introspection schema request to a GraphQL server asynchronously.
-    let sendIntrospectionRequestAsync (connection : GraphQLClientConnection) (serverUrl : string) httpHeaders =
+    let getIntrospectionAsync (connection : GraphQLClientConnection) (serverUrl : string) httpHeaders =
         let sendGet() = async { return! getAsync connection.Client serverUrl }
```
Comment (xperiandri) [inline src/FSharp.Data.GraphQL.Client/GraphQLClient.fs]: Why do we need to wrap that?
```suggestion
        let sendGetAsync() = getAsync connection.Client serverUrl
```

**Review by xperiandri — CHANGES_REQUESTED** [review]

**Review by xperiandri — APPROVED** [review]

**Comment by xperiandri** [comment]
Could you rebase and solve conflicts, please?

**Comment by Birnsen** [comment]
I currently have no access to a Dev-Environmnt and the PR is rather old, so I do not know if the Rebase worked correctly.

**Comment by xperiandri** [comment]
This is handled in the latest versions. Thanks for contribution!

**Pull request closed without merging**